.travis.yml

```diff
@@ -2,6 +2,7 @@ language: node_js
 node_js:
   - "0.10"
   - "0.8"
+  - "io.js"
```

Comment:
It'll be `iojs`. But maybe it's better to specify concrete major+minor versions as in Node, i.e.:
```yaml
```
node_js:
  - "0.10"
  - "0.8"
  - "iojs-1.0"
```